styleguide/derek/incubation/homepage.ejs

```diff
@@ -5,6 +5,8 @@
   'body': 'Redhat names Rackspace North America Managed Service Partner of the Year'
 }) %>
 
+<%- partial("../pattern-components/cust-story-river/cust-story-river.ejs") %>
```

Comment:
While we're leaving the CSS untouched we should still remove the legacy case study HTML from this page as this should show us what the new home page is to look like.